src/ontology/mondo-ingest.Makefile

```diff
@@ -351,14 +351,46 @@ deploy-mondo-ingest:
 
 USE_MONDO_RELEASE=false
 
+# TODO: make sure passes aftr add this line back below cd mondo/src/ontology
+#		make mondo.owl mappings -B MIR=false IMP=false MIR=false &&\
+
 tmp/mondo_repo_built:
 	if [ $(USE_MONDO_RELEASE) = true ]; then wget http://purl.obolibrary.org/obo/mondo.owl -O $@; else cd $(TMPDIR) &&\
 		rm -rf ./mondo/ &&\
 		git clone --depth 1 https://github.com/monarch-initiative/mondo &&\
 		cd mondo/src/ontology &&\
 		make mondo.owl mappings -B MIR=false IMP=false MIR=false &&\
+		latest_hash=$$(git rev-parse origin/master) &&\
 		cd ../../../../ &&\
-		touch $@; fi
+		echo "$$latest_hash" > $@; fi
+
+# TODO
+# explain what does here. explain that -B is just in case tmp/mondo_repo_built exists but not the rpeo
+.PHONY: refresh-mondo-clone
+refresh-mondo-clone:
+	@if [ ! -d tmp/mondo ]; then \
+		$(MAKE) tmp/mondo_repo_built -B; \
+	else \
+		current_hash=$$(cat tmp/mondo_repo_built); \
+		cd tmp/mondo; \
+		git fetch origin; \
+		latest_hash=$$(git rev-parse origin/master); \
+		if [ ! "$$current_hash" = "$$latest_hash" ]; then \
+			cd .. && mv mondo mondo-bak && mv mondo_repo_built mondo_repo_built-bak; \
+			cd .. && $(MAKE) tmp/mondo_repo_built -B; \
+			rm -rf tmp/mondo-bak tmp/mondo_repo_built-bak; \
+		fi; \
+	fi
+
+.PHONY: refresh-mondo-owl
+refresh-mondo-owl: refresh-mondo-clone
+
+.PHONY: refresh-mondo-sssom
+refresh-mondo-sssom: refresh-mondo-clone
```

Comment:
## Unlikely scenario: new commit mid-build
There is an unlikely but possible scenario where, if a new commit goes into `mondo` during the runtime of the build, it's possible that different versions of `mondo.owl` and `mondo.sssom.tsv` could be built at different times in the build, and this could introduce discrepancies.

I can think of a solution to this, where I make the command more intelligent--not only would it check for new commits, but timestamp of the commit would also have to be older than when the build started. And if a new commit was detected, it could throw an error or a warning.

But I think this is probably a low priority. I guess I could make an issue about handling this contingency, perhaps.